src/main/java/com/gmail/goosius/siegewar/listeners/SiegeWarTownEventListener.java

```diff
@@ -406,6 +407,9 @@ public void onTownStatusScreen(TownStatusScreenEvent event) {
 
 							// > Points: +90 / -220
 							out.add(Translation.of("status_town_siege_battle_points", siege.getFormattedAttackerBattlePoints(), siege.getFormattedDefenderBattlePoints()));
+							
+							// > Current Battle Session Reward: 420 points
+							out.add(Translation.of("status_town_siege_winner_gains", SiegeWarBattleSessionUtil.calcPointsForBattleSession(siege)));

```

Comment:
- In regards to displaying the pending balance adjustment on the town screen
  - When winnerTakeAll is true, I don't think it is needed
  - When winnerTakeAll is false, I am not totally sure it is needed. See diagram below and see what you think.
![image](https://user-images.githubusercontent.com/50219223/122942527-2f65dc00-d36e-11eb-9f2a-879283e11ba8.png)

Reply:
I added the bit to the town status as an after-thought, but I think it's worth it. 

Since there's two systems and the players cannot determine which is used I thought it would be good to show exactly what is to be expected from a battle session: otherwise you'll end up with users complaining to their admins that the sessions aren't paying out the point they expected. All of it will have already occurred/finalized and the admins won't be able to show us anything useful when they start to report what their users are passing on to them.

Reply:
- I see what you mean
- Since most SW servers are already using the winner-take all system (from 0.3.0), I suggest that there won't generally be confusion among players there, and that we try to avoid the duplication in that case.
- I also take the point that it would be good for players on the non-default system to see the more complex pending adjustment at a glance.
- Thus to resolve all these issues together, how about we add an if statement around the field, so that it gets displayed for the 'all-points-given' system, but not for the 'winner-take-all' system ?  
- This  avoids the duplication, shows the calculation to players where it is complex, and also allows us to tell at a glance which system is in use